packages/devops/scripts/deployment/setupGoldMaster.sh

```diff
@@ -54,4 +54,5 @@ cd /home/ubuntu
 sudo -Hu ubuntu git clone https://github.com/beyondessential/tupaia.git
 
 # build all packages once using dev to speed up future branch-specific builds
+sudo -Hu ubuntu /home/ubuntu/tupaia/packages/devops/scripts/deployment/fetchEnvVarsFromLastpass.sh gold-master-image-builder
```

Comment:
@edmofro do we actually need the env vars here? If we're just building the packages?

Reply:
Most of them are redundant, but the front end builds do bake in some of the env vars to the build (e.g. the back end server url), plus there's a `SKIP_PREFLIGHT_CHECK` one that's needed. It's a good point though, and it would be more secure to be more specific, and either work those out without going to lastpass, or just get the front end package env vars from lastpass.

Reply:
I'll create a ticket for us to clean up this, since I feel it would be nice for the build to not rely on any `.env` vars, but will leave for now 👍